honeybot/plugins/book_of_day.py

```diff
@@ -0,0 +1,34 @@
+"""
+
+[book_of_day.py]
+Book of the Day Plugin
+[Author]
+Schoberg, UMD
+[About]
+using bookoftheday.org, get the title and link of the book of the day
+[Commands]
+>>> .bookoftheday
+https://bookoftheday.org/fleishman-is-in-trouble-taffy-brodesser-akner/
+"""
+
+import requests
+from bs4 import BeautifulSoup
+
+class Plugin:
+    def __init__(self):
+        pass
+
+    def book(self):
+        headers = {'Accept' : 'application/json'}
+        r = requests.get('https://icanhazdadjoke.com', headers=headers)
+        soup = BeautifulSoup(r.text, 'html.parser')
```

Comment:
hum nice but you have to go to https://bookoftheday.org . If you want to contribute a dad joke plugin, just modify the name and command.

Reply:
ah my mistake. I was using that url while debugging. I just commited with the updated bookoftheday.org